### PR TITLE
implements like functionality

### PR DESCRIPTION
**Pull request by Danwhy**
ref #56

Implements saving likes in postgres, displaying them on resources and limiting likes to one like or dislike per person.

### CODE REVIEW

**Review by samhstn — APPROVED** [review]
I hate to put in criticism here as you've worked so fast and got so much done here, but I'm not sure if I like the ip address being what we're associating with the user's likes/dislikes.
I think storing a cookie in the browser and associating that with a user was better.
I'm happy for this to go in though and we can discuss the alternative in a separate pr.
Great work 👍

File: likes/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```
Comment (nelsonic) [inline likes/tests.py]: @Danwhy if you have a moment, please add a test for the likes functionality. 👍

**Review by nelsonic — COMMENTED** [review]
@Danwhy looks good so far. 👍 
if you have time please add a couple of tests in `likes/tests.py` (_or leave a comment outlining how it can be automatically tested_) thanks!

**Comment by Danwhy** [comment]
@shouston3 I'm not entirely happy with using the IP either, but I think it leaves us open to less abuse than using a cookie, let's try to come up with the best solution here: https://github.com/LDMW/cms/issues/66.

@nelsonic :+1: Will look into getting some tests done today

**Review by nelsonic — APPROVED** [review]
@Danwhy likes code & tests looking good. thanks! 👍

**Pull request merged**